test/UnitTests/MSTestAdapter.UnitTests/Execution/TestAssemblyInfoTests.cs

```diff
@@ -319,7 +319,7 @@ public void RunAssemblyCleanupShouldThrowTheInnerMostExceptionWhenThereAreMultip
 
         var actualErrorMessage = _testAssemblyInfo.RunAssemblyCleanup();
 
-        Verify(actualErrorMessage.StartsWith("Assembly Cleanup method DummyTestClass.AssemblyCleanupMethod failed. Error Message: System.InvalidOperationException: I fail.. StackTrace:", StringComparison.Ordinal));
+        Verify(actualErrorMessage.StartsWith("Assembly Cleanup method DummyTestClass.AssemblyCleanupMethod failed. Error Message: System.TypeInitializationException: The type initializer for 'FailingStaticHelper' threw an exception. ---> System.InvalidOperationException: I fail.. StackTrace:", StringComparison.Ordinal));
```

Comment:
This is strange this stack should not be affected or?

Reply:
AssemblyCleanup is using also the helper methods and that changing the stack

Reply:
This is changed because of the change you did on src/Adapter/MSTest.TestAdapter/Execution/TestAssemblyInfo.cs line 241.